packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -11,34 +12,40 @@ const keyCodes = {
     space: 32,
 };
 
+let menu;
+const onClick = jest.fn();
+const onToggle = jest.fn();
+const onChange = jest.fn();
+
+beforeAll(() => {
+    unmountAll();
```

Comment:
This call should be in a `beforeEach`.

`beforeAll` runs once only for the describe
`beforeEach` runs once before each test case

Reply:
My intention was for the test cases in the `describe` to use the same component. I suppose it may be better to compartmentalize each one.

Reply:
Depending on the same component from one test to another is a bad idea b/c one test can cause other tests to fail.

Reply:
The `unmount` should occur after each test case, not at the very end. Otherwise, there will be side effects between tests.

Reply:
Okay, thanks for all the feedback!